File: examples/python/advanced/serialization/large-payload-errors.py
# Option 1: Reduce data
small_data = {"id": order.id, "status": order.status}
result = context.step(process_order, small_data)

# Option 2: Use summary generator (for map/parallel)
def generate_summary(result):
    return json.dumps({"count": len(result.all)})

config = MapConfig(summary_generator=generate_summary)
result = context.map(process_item, items, config=config)

File: examples/java/advanced/serialization/large-payload-errors.java
// Option 1: Reduce data
var smallData = Map.of("id", order.getId(), "status", order.getStatus());
var result = ctx.step("process-order", Map.class, stepCtx -> processOrder(smallData));

// Option 2: Use summary generator (for map operations)
var config = MapConfig.builder()
    .summaryGenerator(mapResult -> {
        return Map.of("count", mapResult.getAll().size());
    })
    .build();

var result = ctx.map("process-items", items, Map.class, (item, stepCtx) -> process(item), config);
